### PR TITLE
Ember Times sections

### PR DESCRIPTION
**Pull request by jenweber**
Write about:
- The writing workshop
- EmberConf
- Ember Europe
- The annual survey

Also, note to the editors, if there are any spelling errors, wording changes, etc to make, please go ahead and make the changes. You can push to my branch. I will be mostly unavailable the next 2 days.

### CODE REVIEW

**Comment by jenweber** [comment]
@marcoow what do you think about this highlight of the upcoming meetup, for the Ember Times?

File: content/the-ember-times-issue-198.md [hunk 1]
```diff
@@ -14,52 +15,46 @@ tags:
...
+The Ember core teams would love to hear from you!
+  
+The survey is primarily intended for developers who use the Ember JavaScript framework, either for work or personal use, although any developer is welcome to fill out this survey. The survey should take approximately 15 minutes to complete in full, though all questions are optional.
+  
+[Learn more about the survey and fill it out](https://emberjs.com/survey/2022/) by April 7th!
 
 ---
 
-## [🐹 2. Section title in sentence case](section-url)
+## [📽️ 2. Remote Ember.js Europe Meetup, featuring Ember Data](https://twitter.com/embereurope)
```
Comment (marcoow) [inline content/the-ember-times-issue-198.md]:
```suggestion
## [📽️ 2. Hybrid Ember.js Europe Meetup, featuring Ember Data](https://twitter.com/embereurope)
```

The idea behind Ember.js Europe is it's a hybrid event once a quarter from a different location in Europe each time (this time Dublin) as well as remote via Zoom.

**Review by marcoow — COMMENTED** [review]
👍

**Review by jaredgalanis — APPROVED** [review]
🙌

**Pull request merged**